src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -52,14 +52,14 @@ dataverse.oai.server.maxsets=100
 
 # PERSISTENT IDENTIFIER PROVIDERS
```

Comment:
**suggestion**: As the ids might not be the ones folks will use, I suggest we simply delete these defaults here and migrate them into the provider code.

Reply:
OK - removed, and the code will use these as defaults if they aren't set.